cedar-policy-validator/src/schema_file_format.rs

```diff
@@ -252,15 +257,18 @@ pub struct EntityType<N> {
 }
 
 impl EntityType<RawName> {
-    /// Prefix unqualified entity and common type references with the namespace they are in
-    pub fn qualify_type_references(self, ns: Option<&Name>) -> EntityType<Name> {
+    /// (Conditionally) prefix unqualified entity and common type references with the namespace they are in
+    pub fn conditionally_qualify_type_references(
+        self,
+        ns: Option<&Name>,
+    ) -> EntityType<ConditionalName> {
         EntityType {
             member_of_types: self
                 .member_of_types
                 .into_iter()
-                .map(|rname| rname.qualify_with(ns))
+                .map(|rname| rname.conditionally_qualify_with(ns, ReferenceType::Entity)) // REVIEW: Should we allow writing common-types in this position, as long as the common type resolves to an entity type?
```

Comment:
I think it makes sense to punt this change to a future PR, particularly because there are concerns about a breaking change.  (I don't _think_ that anything in this PR is a breaking change -- that it only takes schemas that used to not parse because of attempted reference to an empty-namespace type, and allows them to parse.  There is a regression in error details discussed [here](https://github.com/cedar-policy/cedar/pull/1060#discussion_r1674552526), but I think we should accept that given that #579 is a higher-priority bug.)

Reply:
Sounds fine to me

Reply:
Punted to #1064